doc/snmp/snmp-schema-addition.md

```diff
@@ -66,23 +66,26 @@ In the end this is used to produce */etc/snmp/snmpd.conf*.
          "SNMP_USER_AUTH_TYPE":"MD5|SHA|HMAC-SHA-2",
          "SNMP_USER_ENCRYPTION_TYPE":"DES|AES",
          "SNMP_USER_AUTH_PASSWORD":"<AUTH_PASSWORD_STRING>",
-         "SNMP_USER_ENCRYPTION_PASSWORD":"<ENCRYPTION_PASSWORD_STRING>"
+         "SNMP_USER_ENCRYPTION_PASSWORD":"<ENCRYPTION_PASSWORD_STRING>",
+         "SNMP_USER_PERMISSIONS": "RO|RW"
```

Comment:
Might be better to use singular "SNMP_USER_PERMISSION" as it can only one value, right?

Reply:
Agreed.  Fixed.